### PR TITLE
feat(model): generate ormbfile.yaml automatically

### PR DESCRIPTION
**Pull request by simon-cj**
**What type of PR is this?**
/kind feature
/area model

**What this PR does / why we need it**:
when exec `ormb save` and there are not ormbfile.yaml, generate ormbfile.yaml automatically

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @FogDong 

**Does this PR introduce a user-facing change?**:

```release-note
None
```

### CODE REVIEW

File: pkg/util/util.go [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+	}
+
+	if mxnetFileNum == 2 {
+		return ormbmodel.FormatMXNetParams, nil
+	}
+
+	return ormbmodel.FormatOthers, nil
+}
+
+// WriteORMBFile write ormbfile.yaml if file file is not exist.
```
Comment (FogDong) [inline pkg/util/util.go:75]:
```suggestion
// WriteORMBFile write ormbfile.yaml if file is not exist.
```

File: pkg/saver/saver.go [hunk 2]
```diff
@@ -30,17 +31,32 @@ func New() Interface {
 
 // Save saves the model from the path to the memory.
 func (d Saver) Save(path string) (*model.Model, error) {
+	modelPath := filepath.Join(path, consts.ORMBModelDirectory)
+	ormbfilePath := filepath.Join(path, consts.ORMBfileName)
+
+	if _, err := os.Stat(ormbfilePath); os.IsNotExist(err) {
```
Comment (FogDong) [inline pkg/saver/saver.go:37]: This err need to be handled expect for the not exist

File: pkg/saver/saver.go [hunk 3]
```diff
@@ -30,17 +31,32 @@ func New() Interface {
...
+		format, err := util.InferModelFormat(modelPath)
+		if err != nil {
+			return nil, err
+		}
+
+		if format != "" {
+			err = util.WriteORMBFile(ormbfilePath, format)
```
Comment (FogDong) [inline pkg/saver/saver.go:44]:
```suggestion
			err := util.WriteORMBFile(ormbfilePath, format)
```

File: pkg/util/util.go [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+	"os"
+	"path"
+	"strings"
+
+	"gopkg.in/yaml.v2"
+
+	ormbmodel "github.com/kleveross/ormb/pkg/model"
+)
+
+// InferModelFormat infer model format by file ext.
```
Comment (FogDong) [inline pkg/util/util.go:14]:
```suggestion
// InferModelFormat infers model format by files' ext.
```

**Comment by simon-cj** [comment]
comments  are address, PTAL @FogDong

File: pkg/util/util.go [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+	"os"
+	"path"
+	"strings"
+
+	"gopkg.in/yaml.v2"
+
+	ormbmodel "github.com/kleveross/ormb/pkg/model"
+)
+
+// InferModelFormat InferModelFormat model format by files' ext.
```
Comment (FogDong) [inline pkg/util/util.go:14]:
```suggestion
// InferModelFormat infers model format by files' ext.
```

Comment (simon-cj) [reply pkg/util/util.go:14]: Fixed

**Review by FogDong — COMMENTED** [review]
/lgt,

**Review by FogDong — COMMENTED** [review]
/lgtm

**Pull request merged**